src/main/java/org/schemaspy/output/dot/schemaspy/name/Sized.java

```diff
@@ -0,0 +1,26 @@
+package org.schemaspy.output.dot.schemaspy.name;
+
+/**
+ * Decorates a name based on compactness.
+ */
+public final class Sized implements Name {
```

Comment:
Could you have a look at #466

Reply:
Thank you for providing this link. Could you clarify "look at"? No decision seems to have been reached - do you want me to decide?

Reply:
I should have been more clear. Decision would be great. But since you've spent time thinking about this. You might have additional input or thoughts.

This is not blocking there is a separate issue regarding naming of "size", abridged, unabridged.